direct-auth-samples/src/test/java/info/seleniumcucumber/userStepDefinitions/PasswordRecovery.java

```diff
@@ -108,4 +121,29 @@ public void she_confirms_that_password() {
 		registerPage.confirmNewPasswordInput.click();
 		registerPage.confirmNewPasswordInput.sendKeys("QwErTy@123");
 	}
+
+	@When("she selects \"Forgot Password\"")
+	public void she_selects_forgot_password() {
+		loginPage.forgotPasswordLink.click();
+	}
+
+	@Then("she sees the Password Recovery Page")
+	public void she_sees_the_password_recovery_page() {
+		Assert.assertTrue("URL should ends with \"/forgot-password\"", forgotPasswordPage.getCurrentUrl().endsWith("/forgot-password"));
+		Assert.assertTrue(forgotPasswordPage.forgotPasswordForm.isDisplayed());
+	}
+
+	@When("she inputs an Email that doesn't exist")
+	public void she_inputs_an_email_that_doesnt_exist() {
+		forgotPasswordPage.inputField.sendKeys(EXAMPLE_EMAIL);
+	}
+
+	@Then("she sees a message \"There is no account with the Username mary@unknown.com.\"")
+	public void she_sees_a_message() {
+		WebElement alert = forgotPasswordPage.alertDanger;
+		Assert.assertTrue(alert.isDisplayed());
```

Comment:
It could be simplified so you don't need to import org.openqa.selenium.WebElement
`Assert.assertTrue(forgotPasswordPage.alertDanger.isDisplayed());`

Reply:
Ok, sure. If we want to avoid one extra import, then I can do it :)
But then needs to call alert object twice:
```
		Assert.assertTrue(forgotPasswordPage.alertDanger.isDisplayed());
		String errorMsg = forgotPasswordPage.alertDanger.getText();
```